src/lib/components/ResFormGeneric.svelte

```diff
@@ -257,14 +249,14 @@
 									class="dark:text-white p-0"
 									style="vertical-align:inherit"
 									type="button"
-									on:click={removeBuddyField(bf)}
+									on:click={() => removeBuddyField(bf)}
 									{disabled}
 									tabindex="3"><DeleteIcon svgStyle={'h-6 w-6'} /></button
 								>
 							{/if}
 						</div>
 					</div>
-					{#if bf.matches.length > 0}
+					{#if bf?.matches && bf?.matches?.length > 0}
```

Comment:
What does the '?' at the end of 'bf' and 'matches' do here?

Reply:
its a way to managed possibly undefined properties/keys:

if the next property to ? is undefined, it automatically returns falsy